frontend/app/routes/public/apply/$id/adult-child/children/$childId/confirm-federal-provincial-territorial-benefits.tsx

```diff
@@ -0,0 +1,222 @@
+import { useState } from 'react';
+
+import { redirect, useFetcher } from 'react-router';
+
+import { faChevronLeft, faChevronRight } from '@fortawesome/free-solid-svg-icons';
+import { Trans, useTranslation } from 'react-i18next';
+import { z } from 'zod';
+
+import type { Route } from './+types/confirm-federal-provincial-territorial-benefits';
+
+import { TYPES } from '~/.server/constants';
+import { loadApplyAdultChildState, loadApplyAdultSingleChildState } from '~/.server/routes/helpers/apply-adult-child-route-helpers';
+import { saveApplyState } from '~/.server/routes/helpers/apply-route-helpers';
+import { getFixedT } from '~/.server/utils/locale.utils';
+import { transformFlattenedError } from '~/.server/utils/zod.utils';
+import { Button, ButtonLink } from '~/components/buttons';
+import { CsrfTokenInput } from '~/components/csrf-token-input';
+import { useErrorSummary } from '~/components/error-summary';
+import { InputRadios } from '~/components/input-radios';
+import { LoadingButton } from '~/components/loading-button';
+import { Progress } from '~/components/progress';
+import { pageIds } from '~/page-ids';
+import { getTypedI18nNamespaces } from '~/utils/locale-utils';
+import { mergeMeta } from '~/utils/meta-utils';
+import { getPathById } from '~/utils/route-utils';
+import { getTitleMetaTags } from '~/utils/seo-utils';
+
+const FEDERAL_BENEFITS_CHANGED_OPTION = {
+  no: 'no',
+  yes: 'yes',
+} as const;
+
+export const handle = {
+  i18nNamespaces: getTypedI18nNamespaces('apply-adult-child', 'apply', 'gcweb'),
+  pageIdentifier: pageIds.public.apply.adultChild.confirmFederalProvincialTerritorialBenefits,
+  pageTitleI18nKey: 'apply-adult-child:children.confirm-dental-benefits.title',
+};
+
+export const meta: Route.MetaFunction = mergeMeta(({ data }) => {
+  return getTitleMetaTags(data.meta.title);
```

Comment:
Had to fix this in our v4.0.0 release: We can't expose personal information like names to Adobe Analytics so we have to send the `dcTermsTitle` (rather than the `title`) without the name instead.

```suggestion
  return getTitleMetaTags(data.meta.title, data.meta.dcTermsTitle);
```

Reply:
Added "data.meta.dcTermsTitle"